### PR TITLE
Update laravel echo doc

### PR DESCRIPTION
**Pull request by sacOO7**
- Added link to ably laravel echo doc for configuring ably client-options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,21 +88,7 @@ window.Echo.connector.ably.connection.on(stateChange => {
...
-- You can set additional ably-js [clientOptions](https://ably.com/docs/api/realtime-sdk?lang=javascript#client-options) when creating an `Echo` instance.
-- [Auth specific clientOptions](https://sdk.ably.com/builds/ably/specification/main/features/#AO1) should not be used, since laravel `authEndpoint` is already responsible for token authentication and external auth mechanism is not needed.
-
-```
-    broadcaster: 'ably',
-    echoMessages: true, // self-echo for published message is set to false internally.
-    queueMessages: true, // default: true, maintains queue for messages to be sent.
-    disconnectedRetryTimeout: 15000, // Retry connect after 15 seconds when client gets disconnected
-```
+ Please take a look at [**Ably Laravel Echo Doc**](https://github.com/ably-forks/laravel-echo#readme) for more information related to configuring ably-specific clientOptions and implementing extra features.
```
Comment (owenpearson) [inline README.md:91]:
```suggestion
 Please take a look at the [**Ably Laravel Echo Docs**](https://github.com/ably-forks/laravel-echo#readme) for more information related to configuring ably-specific client options and implementing additional features.
```

File: README.md [hunk 2]
```diff
@@ -222,8 +208,8 @@ i.e. update from [pusher error codes](https://pusher.com/docs/channels/library_a
     })
 ```
 **Note :**
-- AblyPresenceChannel -> `here` method gets called every time a client **joins, updates or leaves** the channel, whereas when using Pusher this is only called once for first client entering the channel.
-- This behaviour follows implementation as per standard [PresenceChannel Interface](https://github.com/laravel/echo/blob/master/src/channel/presence-channel.ts#L10).
+- In `Echo.join().here(members => {})` implementation, members are updated every time a client **joins, updates or leaves** the channel, whereas when using Pusher this is only called once for first client entering the channel.
+- Ably follows this behaviour as per standard [PresenceChannel Interface](https://github.com/laravel/echo/blob/master/src/channel/presence-channel.ts#L10).
```
Comment (owenpearson) [inline README.md:212]:
```suggestion
- In the `Echo.join().here(members => {})` implementation, members are updated every time a client **joins, updates or leaves** the channel, whereas when using Pusher this is only called once for first client entering the channel.
- This behaviour follows the standard Ably [PresenceChannel Interface](https://github.com/laravel/echo/blob/master/src/channel/presence-channel.ts#L10).
```

Comment (sacOO7) [reply README.md:212]: Actually, we are talking about `Ably` following the standard Echo PresenceChannel interface https://github.com/laravel/echo/blob/master/src/channel/presence-channel.ts#L10 interface.

Comment (sacOO7) [reply README.md]: I have updated this accordingly : )

File: README.md (hunk 1 above)
Comment (sacOO7) [reply README.md]: This has been updated as per suggested change.

**Review by owenpearson — APPROVED** [review]
LGTM, thanks!

**Comment by sacOO7** [comment]
Thanks for approving the PR : )

**Pull request merged**